frontend/.env

```diff
@@ -1 +1,8 @@
 VITE_API_BASE_URL="http://localhost:7071/api"
+
+VITE_MSAL_CLIENT_ID="02534232-7aa8-4d22-b92f-c49b69c857d8"
+VITE_MSAL_AUTHORITY="https://login.microsoftonline.com/0d472830-3093-4f90-90aa-bfb7e878d7f2/v2.0"
+VITE_MSAL_REDIRECT_URI="http://localhost:5173/redirect"
+VITE_MSAL_LOGIN_SCOPES="api://a71b9f7e-90e9-465d-bdbc-42a271549219/user_impersonation"
+
+VITE_AZURE_FUNCTION_KEY="c6KDjFtOzGvuoXthFBAadTlGi6H84oNucxTqaX4bz_l7AzFuA2h3vQ=="
```

Comment:
Yes, I added a function key to the version history accidentally, but the key has already been changed & even if it hadn't been, the key is pretty much irrelevant since the functions are behind jwt authentication